workspaces/core/src/select/Select.tsx

```diff
@@ -0,0 +1,152 @@
+import React, {useEffect, useRef, useState} from "react";
+import ChevronDown from "../private/icons/ChrvronDown";
+import ChevronUp from "../private/icons/ChrvronUp";
+import Option from "./private/types/OptionProps";
+import SelectProps from "./private/types/SelectProps";
+
+const Select: React.FC<SelectProps> = ({
```

Comment:
Developers should have a way to access DOM nodes. Please use Refs  for this component